google-cloud-bigtable/acceptance/bigtable/backup_test.rb

```diff
@@ -51,6 +51,10 @@
       _(backup.state).must_equal :READY
       _(backup.creating?).must_equal false
       _(backup.ready?).must_equal true
+      encryption_info = backup.encryption_info
+      _(encryption_info.encryption_type).must_equal :GOOGLE_DEFAULT_ENCRYPTION
```

Comment:
do we have other tests that assert other encryption types for backups?

Reply:
No. I added just enough acceptance coverage to test the surface of the API additions. I would be more than happy to add a full roundtrip test of CMEK. I already added the KMS key to the environment and a test that [creates an instance with the key](https://github.com/googleapis/google-cloud-ruby/blob/7373fa0c45eacd2e631d9141e022e937425e9f73/google-cloud-bigtable/acceptance/bigtable/project_test.rb#L59-L81). Would it be OK to keep that instance around (not delete it after the test run) as we do with the main instance, for faster test runs?

Reply:
@kolea2 Should I add a full roundtrip test of CMEK?

Reply:
I'm working now on a full roundtrip test of CMEK.

Reply:
Apologies for the delay - sounds great, thank you!

Reply:
Added `google-cloud-bigtable/acceptance/bigtable/encryption_test.rb`, below.